app/controllers/podcast_ownerships_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PodcastOwnershipsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @podcast_ownership = PodcastOwnership.new
+  end
+
+  def create
+    @podcast_ownership = PodcastOwnership.new(podcast_ownership_params)
+
+    if @podcast_ownership.save
+      flash[:success] = "Owner created successfully"
+      redirect_to pod_path
+    else
+      flash[:error] = "User is already an owner"
```

Comment:
Use error object on podcast ownership. See: https://guides.rubyonrails.org/active_record_validations.html#valid-questionmark-and-invalid-questionmark